infrastructure/repository/project_impl.go

```diff
@@ -216,25 +217,52 @@ func (r *ProjectRepository) AddProjectMembers(ctx context.Context, projectID uui
 		return repository.ErrInvalidArg
 	}
 
-	// ユーザーの重複チェック
-	projectMembersMap := make(map[uuid.UUID]struct{}, len(projectMembers))
-	for _, v := range projectMembers {
-		if _, ok := projectMembersMap[v.UserID]; ok {
-			return repository.ErrInvalidArg
-		}
-		projectMembersMap[v.UserID] = struct{}{}
-	}
+	projectRaw := new(model.Project)
 
 	// プロジェクトの存在チェック
 	err := r.h.
 		WithContext(ctx).
 		Where(&model.Project{ID: projectID}).
-		First(&model.Project{}).
+		First(&projectRaw).
 		Error
 	if err != nil {
 		return err
 	}
 
+	// 追加するユーザーの期間がプロジェクトの期間内かどうか
+	project := &domain.ProjectDetail{
```

Comment:
このコメントはコードとあってないなので消してよさそう
これの下のチェックがこのコメントに該当してる
```suggestion
	project := &domain.ProjectDetail{
```